app/controllers/proposals_controller.rb

```diff
@@ -99,7 +99,7 @@ def parse_edit_field
   def proposal_params
     params.require(:proposal).permit(:title, {tags: []}, :session_format_id, :track_id, :abstract, :details, :pitch, custom_fields: @event.custom_fields,
                                      comments_attributes: [:body, :proposal_id, :user_id],
-                                     speakers_attributes: [:bio, :user_id, :id])
+                                     speakers_attributes: [:bio, :event_id, :user_id, :id])
```

Comment:
Totally agreed. Mind if I do the same thing with user_id since we already have that available too?